packages/protocol/migrationsConfig.js

```diff
@@ -51,7 +52,7 @@ const DefaultConfig = {
     targetVotingYieldParameters: {
       initial: 0.00016, // (x + 1) ^ 365 = 1.06
       max: 0.0005, // (x + 1) ^ 365 = 1.20
-      adjustmentFactor: 1 / 365,
+      adjustmentFactor: 1 / 3650,
```

Comment:
This feels perfect for the beginning (max. yield decrease per year is now 3.33% and max. yield increase per year is 6.66%). Could be decrease by governance if voting percentage roughly settles at around 66.67%.